tests/autotune/test_tuner.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+import scvi
+
+
+def test_model_tuner_init():
+    model_cls = scvi.model.SCVI
+    scvi.autotune.ModelTuner(model_cls)
+
+
+def test_model_tuner_fit():
+    model_cls = scvi.model.SCVI
+    tuner = scvi.autotune.ModelTuner(model_cls)
+
+    # adata should be setup before passing to `fit`
+    adata = scvi.data.synthetic_iid()
+    with pytest.raises(Exception):
+        tuner.fit(adata)
+
+    model_cls.setup_anndata(adata)
+    results = tuner.fit(adata)
```

Comment:
is there a way to make sure this completes quickly?

Reply:
good point, we can reduce the number of samples for tests